database/youtube/data-structures.json

```
@@ -1,65 +1,74 @@
-[{
+[
+  {
     "name": "Jennys Lectures CSIT",
     "description": "This channel provides free courses for DSA, C, C++, Python etc.",
     "url": "https://www.youtube.com/@JennyslecturesCSIT",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "english"
-},
-{
+  },
+  {
     "name": "CodeHelp - by Babbar",
     "description": "This channel provides free placement specific DSA course in C++.",
     "url": "https://www.youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "Apna College",
     "description": "This channel provides free courses for DSA, Java, C++, C etc.",
     "url": "https://www.youtube.com/@ApnaCollegeOfficial",
     "category": "youtube",
     "subcategory": "dsa",
     "language": "hindi"
-},
-{
+  },
+  {
     "name": "CodeWithHarry",
     "description": "A all in one destination for DSA, Python, C++, JavaScript, Nodejs, Java, PHP & many more.",
```

[Proofreading] I turned A all in one into "An all-in-one" so that users can understand that this YouTube channel offers the content you mentioned. 
```suggestion
    "description": "An all-in-one destination for DSA, Python, C++, JavaScript, Nodejs, Java, PHP, & many more.",
```